src/server/search/search_family.cc

```diff
@@ -183,6 +183,36 @@ optional<search::Schema> ParseSchemaOrReply(DocIndex::DataType type, CmdArgParse
 #pragma GCC diagnostic pop
 #endif
 
+std::string_view ParseField(CmdArgParser* parser) {
+  std::string_view field = parser->Next();
+  if (field.front() == '@') {
+    field.remove_prefix(1);  // remove leading @ if exists
+  }
+  return field;
+}
+
+std::optional<std::string_view> ParseFieldWithAtSign(CmdArgParser* parser) {
+  std::string_view field = parser->Next();
+  if (field.front() != '@') {
+    return std::nullopt;  // if we expect @, but it's not there, return nullopt
+  }
+  field.remove_prefix(1);  // remove leading @
+  return field;
+}
+
+void ParseLoadFields(CmdArgParser* parser, std::optional<SearchFieldsList>& load_fields) {
```

Comment:
before Roman finds it: nit mutable values are passed by pointer 🙂

Reply:
Yes, it was temporary

Reply:
😄